datalab/datalab_session/analysis/line_profile.py

```diff
@@ -29,13 +30,12 @@ def line_profile(input: dict):
     start_coords = [start_sky_coord.ra.deg, start_sky_coord.dec.deg]
     end_coords = [end_sky_coord.ra.deg, end_sky_coord.dec.deg]
 
-  except Exception as e:
-    print(e)
+  except WcsError as e:
+    print(error, e)
     error = f'{input["basename"]} does not have a valid WCS header'
     # if theres no valid WCS solution then we default to using pixscale to calculate the angle, and no coordinates
     start_coords = None
     end_coords = None
     arcsec_angle = len(line_profile) * sci_hdu.header["PIXSCALE"]
 
   return {"line_profile": line_profile, "arcsec": arcsec_angle, "start_coords": start_coords, "end_coords": end_coords}
```

Comment:
This file still missing newline :)

Reply:
I think its good now.. that setting is very helpful